code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -849,3 +849,50 @@
 		M.adjustFireLoss(-2*REM, 0)
 	M.adjustStaminaLoss(-5*REM, 0)
 	..()
+
+/datum/reagent/consumable/cheese_spread
+	name = "Cheese Spread"
+	description = "I cant believe its not cheese!"
+	color = "#FBDB65"
+	nutriment_factor = 2 * REAGENTS_METABOLISM
+	taste_mult = 2
+	taste_description = "cheese"
+
+/datum/reagent/waterpowder
```

Comment:
I'm questioning the logic of dehydrated water.

Reply:
LMAO

Reply:
Um contact valorium about this one

Reply:
Dehydrated water isn't in the design doc? Why do you have it in here